src/front/js/pages/landing.js

```diff
@@ -0,0 +1,22 @@
+import React from "react";
+
+import Testimonials from "../misc/Testimonials";
+import Pricing from "../misc/Pricing";
+import Header from "../misc/Header";
+
+const Home = () => {
```

Comment:
I suppose you did this file for testing somethings, since it's very similar to your `home.js`...

Is there another reason why you would want to keep it here, or could we delete it already?

Reply:
exactly in fact im going to discard home page an use this landing only as root if thats ok